string file_path
---
sensor_msgs/Image grayscale_image
